emoji/src/main/java/com/vanniktech/emoji/EmojiView.java

```diff
@@ -39,20 +42,35 @@
 
   public EmojiView(final Context context, final OnEmojiClickListener onEmojiClickListener,
             final OnEmojiLongClickListener onEmojiLongClickListener, @NonNull final RecentEmoji recentEmoji,
-            @NonNull final VariantEmoji variantManager) {
+            @NonNull final VariantEmoji variantManager, int backgroundColor, int iconColor, int dividerColor) {
     super(context);
 
     View.inflate(context, R.layout.emoji_view, this);
 
     setOrientation(VERTICAL);
-    setBackgroundColor(ContextCompat.getColor(context, R.color.emoji_background));
+    if (backgroundColor != 0)
+      setBackgroundColor(backgroundColor);
+    else
+      setBackgroundColor(ContextCompat.getColor(context, R.color.emoji_background));
+
+
+    if (iconColor != 0)
+      themeIconColor = iconColor;
+    else
+      themeIconColor=  ContextCompat.getColor(context, R.color.emoji_icons);
+
 
-    themeIconColor = ContextCompat.getColor(context, R.color.emoji_icons);
     final TypedValue value = new TypedValue();
     context.getTheme().resolveAttribute(R.attr.colorAccent, value, true);
     themeAccentColor = value.data;
 
     final ViewPager emojisPager = findViewById(R.id.emojis_pager);
+    final View emojiDivider = findViewById(R.id.emoji_divider);
+    if (dividerColor != 0)
+      emojiDivider.setBackgroundColor(dividerColor);
+    else
```

Comment:
Braces are missing

Reply:
is it important to have braces?  
You mean that?
 if (dividerColor != 0){
      emojiDivider.setBackgroundColor(dividerColor);
   } else{
      emojiDivider.setBackgroundColor(getResources().getColor(R.color.emoji_divider));
}